applications/crossbar/src/crossbar_doc.erl

```diff
@@ -409,20 +409,21 @@ load_view(#load_view_params{dbs = []
     end;
 load_view(#load_view_params{page_size = PageSize
                            ,context = Context
-                           ,should_paginate = true
+                           ,should_paginate = 'true'
                            })
-  when is_integer(PageSize), PageSize =< 0 ->
+  when is_integer(PageSize)
+       andalso PageSize =< 0 ->
     lager:debug("page_size exhausted: ~p", [PageSize]),
-    case success =:= cb_context:resp_status(Context) of
-        false -> Context;
-        true -> handle_datamgr_success(cb_context:doc(Context), Context)
+    case 'success' =:= cb_context:resp_status(Context) of
+        'false' -> Context;
+        'true' -> handle_datamgr_success(cb_context:doc(Context), Context)
     end;
 load_view(#load_view_params{view = View
                            ,view_options = Options
                            ,context = Context
                            ,start_key = StartKey
                            ,page_size = PageSize
-                           ,dbs = [Db|Dbs]
+                           ,dbs = [Db|_]=Dbs
```

Comment:
I don't think that's right (wrt L472).
Have you tried this with the ledgers API? I think this will return duplicates.